docs/bidders/kidoz.md

```diff
@@ -0,0 +1,9 @@
+# Kidoz Bidder
+
+Kidoz is exclusively for Mobile app coppa compatible ads, 100% kid relevant and appropriate.
```

Comment:
Could we slightly fix the capitalization? 
`Kidoz is exclusively for mobile app COPPA compatible`

Reply:
done

Reply:
Thanks.